packages/performance-profiler-plugin/package.json

```diff
@@ -22,17 +22,18 @@
     "dist"
   ],
   "main": "dist/index.js",
-  "types": "dist/typings/index.d.ts",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "build": "cross-env NODE_ENV=development webpack",
-    "build:prod": "cross-env NODE_ENV=production webpack",
-    "build:watch": "cross-env NODE_ENV=development webpack --watch",
+    "build": "tsc -b .",
```

Comment:
I think the change here will stop working outside local development because we switched away from webpack. I'm pretty sure the frontend won't get loaded into the app since its using webpack raw-loader. I "think" it's working locally because the `demo` app is actually doing this for us. Probably need to convert this back to a webpack build for now.